src/App.tsx

```diff
@@ -8,7 +8,7 @@ import { LogBox } from 'react-native'
 import CodePush from 'react-native-code-push'
 
 // if __DEV__ import if you want to debug
-// import './why-did-you-render'
+import './why-did-you-render'
```

Comment:
tu peux le re-commenter ?

Reply:
je ne comprends pas le besoin de le commenter alors qu'il est conditionner

https://github.com/pass-culture/pass-culture-app-native/blob/cf61dff95583d6ecc957b16b985de162dc719988/src/why-did-you-render.js#L6

Reply:
il me semble que ça se retrouve dans le build

Reply:
il n'était pas importé comme la documentation le dit de le faire https://github.com/welldone-software/why-did-you-render#setup

j'ai fait un commit pour faire comme le dit de le faire la documentation https://github.com/pass-culture/pass-culture-app-native/pull/2333/commits/010adc4aca2f324c94a7cc810f8aef34b282fdc0

si tu préfères, je peux supprimer le commit et commenter l'import comme avant

Reply:
oui il y a un commentaire au dessus `if __DEV__ import if you want to debug` pour dire: si tu veux mesurer les perfs, tu peux décommenter la ligne en dessous.
Je préfère en effet qu'on le garde commenté et qu'on dé-commente cette ligne en local si on veut faire des améliorations de perf, pour être sûr que ça se retrouve pas sur le build de prod, même si c'est derrière un env. Au moins au augmente pas la taille du bundle